internal/pkg/util/uri/uri.go

```diff
@@ -15,6 +15,10 @@ const (
 	Library = "library"
 	// Shub is the keyword for a shub ref
 	Shub = "shub"
+	// Http is the keyword for http ref
+	Http = "http"
+	// Https is the keyword for https ref
+	Https = "https"
```

Comment:
Can you change these constants from `Http(s)` to `HTTP(S)` to make the go linter happy?

Reply:
updated in new commit  58b0ac6